src/omnibase_core/models/container/model_onex_container.py

```
@@ -282,7 +282,7 @@ async def get_service_async(
                     },
                 )
 
-                typed_service: T = cast(T, service_instance)
+                typed_service: T = cast("T", service_instance)
                 return typed_service
```

Ruff (0.14.7)

Consider moving this statement to an `else` block

(TRY300)